apps/meteor/server/models/raw/LivechatVisitors.ts

```diff
@@ -41,17 +41,21 @@ export class LivechatVisitorsRaw extends BaseRaw<ILivechatVisitor> implements IL
 		];
 	}
 
-	findOneVisitorByPhone(phone: string): Promise<ILivechatVisitor | null> {
+	findOneVisitorByPhone(phone: string, sourceFilter?: Filter<ILivechatVisitor>): Promise<ILivechatVisitor | null> {
```

Comment:
I think it would be better to create a new method for this making it explicit we are filtering it by source, instead of reusing this one and making the filter optional.